metals/src/main/scala/scala/meta/internal/builds/SbtBuildTool.scala

```diff
@@ -132,13 +187,35 @@ case class SbtBuildTool(
     }
   }
 
-  override def toString: String = "sbt"
+  override def toString: String = SbtBuildTool.name
 
-  def executableName = "sbt"
+  def executableName = SbtBuildTool.name
 }
 
 object SbtBuildTool {
 
+  val name = "sbt"
+
+  // Note: (ckipp01) first version was actually 1.4.0, but there is some issues
+  // with bsp discovery with it. So we instead are going with 1.4.1 being the
+  // first actually supported sbt version for bsp support in Metals.
+  val firstVersionWithBsp = "1.4.1"
+
+  def writeSbtBspPlugin(workspace: AbsolutePath): Unit = {
+    // TODO-BSP right now we just check to see if the file is there, but rather
```

Comment:
I'll probably address this after we publish the plugin since I'll have to change this around a bit then.

Reply:
Addressed now that it's an actual plugin.